tests/merch.test.ts

```diff
@@ -564,4 +569,81 @@ describe('merch order pickup events', () => {
     expect(persistedPickupEvent.orders).toHaveLength(1);
     expect(persistedPickupEvent.orders[0]).toStrictEqual(persistedOrder);
   });
+
+  test('placing an order with a pickup event that has reached the order limit fails', async () => {
+    const conn = await DatabaseConnection.get();
+    const member = UserFactory.fake({
+      points: 100,
+    });
```

Comment:
nit: It should be fine to make this one-line since it's only one property we're passing

Reply:
Resolved in 895837e.